tests/client/client_test.py

```diff
@@ -1,1810 +1,551 @@
-            )
-        ),
-    )
+def test_get_req_id_by_tx_hash_success(pyband_client):
+    reqId = pyband_client.get_request_id_by_tx_hash(b"txhash")
+    assert reqId == "154966"
+
+
+def test_get_req_id_by_tx_hash_invalid_input(pyband_client):
```

Comment:
What's difference error between invalid tx hash/ transaction failed / tx hash that didn't contain request?

Reply:
Added, but not sure about the response of tx hash that doesn't contain a request 🦧